### DISCUSSION TITLE
CLI for adjusting brightness?

### DISCUSSION THREAD

**Discussion by lavinnyv** [MonitorControl/MonitorControl#826] [Q&A]
looking to ultimately do this remotely over ssh or something. but just being able to adjust it locally in the command line would be a great asset for me.

**Comment by waydabber** [comment]
Hi @lavinnyv - if you need a simple CLI tool, you can use [m1ddc](https://github.com/waydabber/m1ddc) on Apple Silicon or [ddcctl](https://github.com/kfix/ddcctl) for Intel. An other great option is Lunar (@alin23) which has full CLI built-in.

We might add CLI to monitorcontrol in the future.

**Reply by lavinnyv** [reply]
thank you

**Reply by jonahclarsen** [reply]
Thanks for sharing. I was looking for a similar thing to automate my monitor's brightness at morning/night.

Unfortunately, m1ddc still leaves an issue when used in conjunction with MonitorControl - when I set the brightness/contrast from m1ddc and then change it using keyboard shortcuts from MonitorControl, it reverts to the old settings (i.e. MonitorControl doesn't realize that m1ddc changed the settings). Using the "attempt to read" setting doesn't work for me, even after restarting MonitorControl - I'm assuming my monitor doesn't support that feature.

I know CLI support might be a big ask, but I wanted to add my voice to the mix to say that it would be a much appreciated feature. Either way, thank you for such a useful tool.
